boards/sparc/bm3823/xx3823/configs/nsh/defconfig

```diff
@@ -25,29 +25,21 @@ CONFIG_DEBUG_SYMBOLS=y
 CONFIG_DEBUG_WARN=y
 CONFIG_DEFAULT_SMALL=y
 CONFIG_DISABLE_MOUNTPOINT=y
-CONFIG_DISABLE_MQUEUE=y
```

Comment:
Why are the options from defconfig removed?

Reply:
since it's default setting if CONFIG_DEFAULT_SMALL equals y